packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-validation.spec.ts

```diff
@@ -114,258 +113,266 @@ describe('DbasS API Error Handling', () => {
+      .findByLabel('Node Type')
+      .should('be.visible')
+      .type(`${nodeType}{enter}`);
+
+    // Wait for the metric definitions API call to resolve.
+    cy.wait('@getMetricDefinitions');
+    cy.get('[data-qa-error-msg="true"]')
+      .should('be.visible')
+      .and('have.text', 'Error loading the definitions of metrics.');
+  });
+
+  it('should return error response when fetching services API request', () => {
+    cy.intercept('GET', apiMatcher(`/monitor/services`), {
+      statusCode: 400,
+      body: {
+        errors: [{ reason: 'Bad Request' }],
+      },
+    }).as('fetchServices');
+    cy.visitWithLogin('monitor/cloudpulse');
+
+    // Wait for the API call to complete and capture the response.
+    cy.wait('@fetchServices');
+
+    cy.get('[data-qa-textfield-error-text="Dashboard"]')
+      .should('be.visible')
+      .invoke('text')
+      .then((text) => {
+        expect(text).to.equal('Failed to fetch the services.');
+      });
+  });
+  it('should return error response when fetching token api request', () => {
+    cy.intercept('POST', apiMatcher(`/monitor/services/${serviceType}/token`), {
+      statusCode: 400,
+      body: {
+        errors: [{ reason: 'Bad Request' }],
+      },
+    }).as('fetchToken');
+
+    cy.visitWithLogin('monitor/cloudpulse');
+    //  Wait for both the fetch services and fetch dashboard API calls to complete.
+    cy.wait(['@fetchServices', '@fetchDashboard']);
+
+    // Selecting a dashboard from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Dashboard')
+      .should('be.visible')
+      .type(`${dashboardName}{enter}`)
+      .should('be.visible');
+
+    //  Select a Database Engine from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Database Engine')
+      .should('be.visible')
+      .type(`${engine}{enter}`)
+      .should('be.visible');
+
+    //  Select a region from the dropdown.
+    ui.regionSelect.find().click().type(`${region}{enter}`);
+
+    // Select a resource (Database Clusters) from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Database Clusters')
+      .should('be.visible')
+      .type(`${clusterName}{enter}`)
+      .click();
+    cy.findByText(clusterName).should('be.visible');
+
+    // Select a Node from the autocomplete input.
+    ui.autocomplete
+      .findByLabel('Node Type')
+      .should('be.visible')
+      .type(`${nodeType}{enter}`);
+
+    cy.get('[data-qa-error-msg="true"]')
+      .should('be.visible')
+      .and('have.text', 'Failed to get the authentication token.');
+  });
 
-    it(`should return ${statusCode} error message when the Regions API request fails`, () => {
-      cy.intercept('GET', apiMatcher(`regions*`), {
-        statusCode: statusCode, // Use the status code defined in the test
+  it('should return error response when fetching Dashboards API Request', () => {
+    mockGetCloudPulseServices(serviceType).as('fetchServices');
+    cy.intercept(
```

Comment:
FYI, as per our discussions, I believe in other places also, the intercept is handled within the test cases, rather putting in a intercepts.ts file.

cc- @kmuddapo , @nikhagra-akamai